tools/ports/__init__.py

```diff
@@ -36,6 +36,7 @@
 
 logger = logging.getLogger('ports')
 
+build_deferred = False
```

Comment:
This global variable is ugly but it allows this approach to work without having to edit all the ports definition files. The alternative would be to thread the deferred argument through the `get()` of each port and its calls of `ports.build_port`.